source/tutorials/sample-classifier.rst

```diff
@@ -0,0 +1,109 @@
+
+We will download and create several files, so first create a working directory.
+
+.. command-block::
+   :no-exec:
+
+   mkdir sample-classifier-tutorial
+   cd sample-classifier-tutorial
+
+Predicting categorical sample data
+----------------------------------
+
+Supervised learning classifiers predict categorical sample data as a function of feature composition. For example, we may use a classifier to diagnose or predict disease susceptibility based on stool microbiome composition, or predict sample type as a function of the sequence variants, microbial taxa, or metabolites detected in a sample. In this tutorial, we will use the `moving pictures tutorial data`_ to train a classifier that predicts the body site from which a sample was collected. Download the feature table and sample metadata with the following links:
+
+.. download::
+   :url: https://data.qiime2.org/2017.7/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: moving-pictures-sample-metadata.tsv
+
+.. download::
+   :url: https://docs.qiime2.org/2017.7/data/tutorials/moving-pictures/table.qza
+   :saveas: moving-pictures-table.qza
+
+Next, we will attempt to predict from which body site each sample originates as a function of sequence variant composition.
+
+.. command-block::
+
+   qiime sample-classifier classify-samples \
+    --i-table moving-pictures-table.qza \
+    --m-metadata-file moving-pictures-sample-metadata.tsv \
+    --m-metadata-category BodySite \
+    --o-visualization moving-pictures-BodySite \
+    --p-optimize-feature-selection \
+    --p-parameter-tuning \
+    --p-estimator RandomForestClassifier \
+    --p-n-estimators 100
+
+The visualization produced by this command presents classification accuracy results in the form of a confusion matrix. This matrix indicates how frequently a sample is classified with to the correct class vs. all other classes. The confusion matrix is displayed at the top of the visualization in the form of a heatmap, and below that as a table containing overall accuracy (the fraction of times that test samples are assigned the correct class). 
+
+.. question::
+   What other metadata can we predict with ``classify-samples``? Take a look at the metadata categories in the ``sample-metadata`` and try some other categories. Not all metadata can be easily learned by the classifier! 
+
+
+If ``--p-optimize-feature-selection`` is enabled, the visualization will also display a recursive feature extraction plot, which illustrates how model accuracy changes as a function of feature count. The combination of features that maximize accuracy are automatically selected for the final model, which is used for sample prediction results that are displayed in the visualization. A list of the features chosen, and their relative importances, will be displayed at the bottom of the visualization. Features with higher importance scores are more important for distinguishing each class.
```

Comment:
Out of curiosity, is cross-validation used here?  May be worthwhile to mention that if it is.